input/fsh/examples/CBTN_example.fsh

```diff
@@ -350,10 +350,10 @@ InstanceOf: NcpiFamilyRelationship
 Title: "An example family relationship based on data from CBTN"
 Usage: #example
 Description: "An example family relationship based on data from CBTN."
-* subject = Reference(PT-006SP660)
-* focus = Reference(PT-006SP675)
-* code = $family-role-code#MTH "mother"
```

Comment:
I swapped the "patient" and "relative" to use the preferred child encoding rather than "mother."